package.json

```diff
@@ -12,6 +12,10 @@
         "branch": "",
         "SHA": ""
     },
+    "dependencies": {
```

Comment:
Yes, it's possible to move the node_modules to the extension directory, do you want it to be committed there?